face/infra/roc-v3/src/main/java/com/simprints/face/infra/rocv3/detection/RocV3Detector.kt

```diff
@@ -50,7 +50,7 @@ class RocV3Detector @Inject constructor() : FaceDetector {
         val template = roc3.new_uint8_t_array(roc3.ROC_FACE_FAST_FV_SIZE.toInt())
         val yaw = roc3.new_float()
         val quality = roc3.new_float()
-        val face = if (isFaceDetected(grayImage, detection)) {
+        val face = if (isFaceDetected(coloredImage, detection)) {
```

Comment:
@luhmirin-s  Why  you changed this line?

Reply:
The accurate method uses color images as per documentation.